src/3d/qgs3daxis.cpp

```diff
@@ -438,41 +477,53 @@ void Qgs3DAxis::createAxisScene()
 
     updateAxisLabelPosition();
   }
+
+  mParentWindow->requestUpdate();
```

Comment:
I have not seen the QWindow::requestUpdate() calls anywhere in QGIS code so far - do we really need them for 3D axis for some reason or are they here "just to be sure" ?